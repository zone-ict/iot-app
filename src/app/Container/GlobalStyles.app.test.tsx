import React from 'react';
import renderer from 'react-test-renderer';
import GlobalStyles from './GlobalStyles.app';

describe('GlobalStyles component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<GlobalStyles />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
