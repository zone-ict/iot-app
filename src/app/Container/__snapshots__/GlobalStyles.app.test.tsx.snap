// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyles component should render correctly 1`] = `null`;
